lib/codit/population/networks/home_locations.py

```diff
@@ -8,52 +8,81 @@
-                 "semidetached_house",
-                 "static_caravan",
-                 "terrace"]
+                  "bungalow",
+                  "cabin",
+                  "detached",
+                  "dormitory",
+                  "farm",
+                  "ger",
+                  "hotel",
+                  "house",
+                  "houseboat",
+                  "residential",
+                  "semidetached_house",
+                  "static_caravan",
+                  "terrace"]
+
+multiple_households_building_types = [
+    "apartments",
+    "residential",
+    "terrace"
+]
 
 
 class Home:
-    def __init__(self, lon=0.0, lat=0.0, accommodation_type=''):
+    def __init__(self, lon=None, lat=None, accommodation_type='', ward_code='', ward_name=''):
         self.coordinate = {'lon': lon, 'lat': lat}
         self.type = accommodation_type
+        self.ward = Ward(ward_code, ward_name)
 
 
 def get_coords(csvfilename):
     """
     Get coordinates and building_type from previous queries to OpenStreetMap
     :param csvfilename: csv file that stores the coordinates and building_type
-    :return: return list of [float(coord['lon']), float(coord['lat']), str(coord['building_type'])]
+    :return: return list of [float(coord['lon']), float(coord['lat']), str(coord['building_type']),
+                                                    str(coord['ward_name']), str(coord['ward_code'])]
     """
-    coords  = []
+    coords = []
     with open(csvfilename, 'r') as csv_coords_f:
         coords_rd = csv.DictReader(csv_coords_f)
-        coords += [[float(coord['lon']), float(coord['lat']), str(coord['building_type'])]
-               for coord in coords_rd]
+        coords += [
+            [float(coord['lon']), float(coord['lat']), coord['building_type'].strip(), coord['ward_name'].strip(),
+             coord['ward_code'].strip()] for coord in coords_rd]
         return coords
 
 
+def get_population_wards(csvfilename):
+    """
+    Get Population of each ward in a LA
+    :param csvfilename: csv file that stores the ward_name ward_code and population
+    :return: return list of [str(pop_ward['ward_code']), str(pop_ward['ward_name']), int(pop_ward['population'])]
+    """
+    population_wards = []
+    with open(csvfilename, 'r') as csv_pop_ward_f:
+        pop_ward_rd = csv.DictReader(csv_pop_ward_f)
+        population_wards += [
+            [str(pop_ward['ward_code']).strip(), str(pop_ward['ward_name']).strip(), int(pop_ward['population'])]
+            for pop_ward in pop_ward_rd]
+        return population_wards
+
+
 def count_coords_for_types(coords):
     """
     Count number of coordinates for each building-type from coordinates-building_type list
-    The coordinates list requested from Openstreetmap is organized as a list of ['lon','lat','building_type'].
+    The coordinates list requested from Openstreetmap is organized as a list of ['lon','lat','building_type','ward_name','ward_code'].
```

Comment:
Which provider populates the Ward Name and Ward Code here? (am assuming this is not openstreetmap)